src/compile/data/aggregate.ts

```diff
@@ -1,23 +1,20 @@
 import {AggregateOp} from '../../aggregate';
-import {isScaleChannel} from '../../channel';
+import {Channel, isScaleChannel} from '../../channel';
 import {field, FieldDef} from '../../fielddef';
 import * as log from '../../log';
 import {SummarizeTransform} from '../../transform';
-import {NOMINAL, ORDINAL} from '../../type';
 import {Dict, differ, duplicate, extend, keys, StringSet} from '../../util';
 import {VgAggregateTransform} from '../../vega.schema';
 import {UnitModel} from './../unit';
+import {binRequiresRange} from './bin';
```

Comment:
It's weird that aggregate node depends on bin node, should binRequiresRange be somewhere else such as common?

Reply:
Done